src/cli/init.rs

```diff
@@ -62,6 +64,13 @@ const PYROJECT_TEMPLATE: &str = r#"
 name = "{{ name }}"
 channels = [{%- if channels %}"{{ channels|join("\", \"") }}"{%- endif %}]
 platforms = ["{{ platforms|join("\", \"") }}"]
```

Comment:
I am not sure the `join` is needed at all. The following seems to work alright
```suggestion
platforms = {{platforms}}
```

Reply:
Makes sense! If it works lets change it.

Reply:
Done in https://github.com/prefix-dev/pixi/pull/1077/commits/bbe7d9e0309fcd2051ecdf10402904a32754587e